orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationAiModelParameters.java

```diff
@@ -0,0 +1,86 @@
+package com.sap.ai.sdk.orchestration;
+
+import java.util.Map;
+import javax.annotation.Nonnull;
+
+/** Helper interface to define typed parameters. */
+@FunctionalInterface
+public interface OrchestrationAiModelParameters {
+  /**
+   * Get the parameters.
+   *
+   * @return the parameters.
+   */
+  @Nonnull
+  Map<String, Object> getParams();
+
+  /** GPT model parameters. */
+  interface GPT extends OrchestrationAiModelParameters {
```

Comment:
[q] Are these GPT specific (model) parameters or Azure specific (platform/Rest API)?

Reply:
After talking to Orchestration team, I was made aware, these parameters are expected to work with all LLM providers (today). They are mapping the above OpenAI related parameter to all supported providers.